tilavarauspalvelu/admin/email_template/forms.py

```diff
@@ -161,30 +189,30 @@ class ReservationBaseForm(
 
 class ApplicationHandledEmailTemplateTesterForm(BaseEmailTemplateForm):
     def to_context(self) -> EmailContext:
-        return get_context_for_application_handled(language=self.cleaned_data["language"])
+        return get_context_for_application_handled(**super().get_context_params())
```

Comment:
You should just call `self.get_context_params()` in all of these implementations since you might want to modify the context params in the class itself, and this would now not enable that. Generally, you'd only want to use `super()` to call a the same method of a superclass, not any other method.

Reply:
Yep, you're right, not sure what I was thinking writing it like this 😆